release-notes/6.0/preview/api-diff/preview7/.Net/6.0.0-preview7_System.IO.md

```diff
@@ -0,0 +1,34 @@
+# System.IO
+
+``` diff
+ namespace System.IO {
+     public static class Directory {
++        public static FileSystemInfo CreateSymbolicLink(string path, string pathToTarget);
++        public static FileSystemInfo ResolveLinkTarget(string linkPath, bool returnFinalTarget);
+     }
+     public static class File {
++        public static FileSystemInfo CreateSymbolicLink(string path, string pathToTarget);
++        public static FileSystemInfo ResolveLinkTarget(string linkPath, bool returnFinalTarget);
+     }
+     public abstract class FileSystemInfo : MarshalByRefObject, ISerializable {
++        public string LinkTarget { get; }
++        public void CreateAsSymbolicLink(string pathToTarget);
++        public FileSystemInfo ResolveLinkTarget(bool returnFinalTarget);
+     }
+     public static class RandomAccess {
+-        public static long Write(SafeFileHandle handle, IReadOnlyList<ReadOnlyMemory<byte>> buffers, long fileOffset);
+
```

Comment:
Nit: the blank lines here are strange... they appear to be an attempt at grouping, but if so they're grouping incorrectly.